src/contrast/APCA.js

```diff
@@ -54,11 +55,15 @@ export default function contrastAPCA (background, foreground) {
 
 	// Calculates "screen luminance" with non-standard simple gamma EOTF
 	// weights should be from CSS Color 4, not the ones here which are via Myndex and copied from Lindbloom
-	[R, G, B] = foreground.coords;
+	[R, G, B] = foreground.coords.map(c => {
+		return isNone(c) ? 0 : c;
+	});
 	let lumTxt = linearize(R) * 0.2126729 + linearize(G) * 0.7151522 + linearize(B) * 0.0721750;
 
 	background = to(background, "srgb");
-	[R, G, B] = background.coords;
+	[R, G, B] = background.coords.map(c => {
+		return isNone(c) ? 0 : c;
```

Comment:
Not a blocker, but I think we need something higher level than this, i.e. a function to drop `none`s. If that function does not mutate objects but returns new ones, it will also fix the serialization problem. Can you open a new issue for the API design discussion?

Reply:
> Not a blocker, but I think we need something higher level than this

I generally agree.

> Can you open a new issue for the API design discussion?

Yep. Issue created: https://github.com/color-js/color.js/issues/479